client/src/state/webauthnAvailableReducer.js

```diff
@@ -0,0 +1,26 @@
+/* global window */
+/**
+ * Adds information about whether WebAuthn is supported by the current browser into the Redux
+ * store, so the application can react accordingly.
+ *
+ * @param {object} state
+ * @returns {object}
+ */
+export default function webauthnAvailableReducer(state = {}) {
+  // See https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API#Interfaces
+  const isAvailable = typeof window.AuthenticatorResponse !== 'undefined';
+
+  // Only provide a "mfaRegister" state override if unavailable
+  const webAuthnAvailable = isAvailable ? {} : {
```

Comment:
Feels a little weird to me for the two isAvailable states to be `undefined` or `false`, can you give a little context on why you took this approach?

Reply:
Yes, undefined is if you have nothing to say from the frontend, because if you say true it will override a backend false. If you say nothing a backend false will be honoured. This is in the MFA module docs, and yeah that PR is required first